tests/integration/concrete/lockup-base/withdraw-multiple/withdrawMultiple.tree

```diff
@@ -8,17 +8,21 @@ WithdrawMultiple_Integration_Concrete_Test
       ├── when zero array length
       │  └── it should do nothing
       └── when non zero array length
-         ├── given atleast one null stream
-         │  └── it should revert
-         └── given no null streams
-            ├── given atleast one DEPLETED stream
-            │  └── it should revert
-            └── given no DEPLETED streams
-               ├── when atleast one zero amount
-               │  └── it should revert
+         ├── when one null stream
+         │  ├── it should emit {InvalidWithdrawalInWithdrawMultiple} event for the null stream
+         │  └── it should emit 2 {WithdrawFromLockupStream} events
+         └── when no null streams
+            ├── when one DEPLETED stream
+            │  ├── it should emit {InvalidWithdrawalInWithdrawMultiple} event for the depleted stream
+            │  └── it should emit 2 {WithdrawFromLockupStream} events
+            └── when no DEPLETED streams
+               ├── when withdrawing zero amounts
+               │  ├── it should emit 1 {WithdrawFromLockupStream} event for first stream
```

Comment:
i don't think we need to test each particular case when it reverts

i would rather have this:

```tree
├── when at least one ID reverts
│  ├── it should emit {InvalidWithdrawalInWithdrawMultiple} event for that ID
│  └── it should wihtdraw from the rest IDs
└── when no IDs revert
```

Reply:
Agree. I kept all those tests in the beginning to check all possible scenarios. But will do as per your suggestion.